sections/admin/home.jsx

```diff
@@ -44,6 +44,9 @@ const AdminHome = () => {
     return "old";
   };
 
+  const appList = useRef();
+  const isFirst = useRef(true);
+
```

Comment:
If there is no appointments appList will crash the app
![image](https://user-images.githubusercontent.com/84677177/195432919-ff2f438c-45e2-492c-928d-06c0869af407.png)
![image](https://user-images.githubusercontent.com/84677177/195433008-8570e5e0-63d3-495d-b527-e9a678bfdb9e.png)
Because this ref will be added only if `appointments.length > 0 || appointmentsByDate.length` is `true`

Reply:
Hi, fixed it.